Modules/Office365DSC/DSCResources/MSFT_EXOAntiPhishPolicy/MSFT_EXOAntiPhishPolicy.schema.mof

```diff
@@ -0,0 +1,31 @@
+[ClassVersion("1.0.0.0"), FriendlyName("EXOAntiPhishPolicy")]
+class MSFT_EXOAntiPhishPolicy : OMI_BaseResource
+{
+    [Write, ValueMap{"Present","Absent"}, Values{"Present","Absent"}] String Ensure;
```

Comment:
Please provide description for every parameter since the Wiki content is automatically extracted from it.

Reply:
Descriptions have been populated: